three.js/vendor/jsartoolkit5/js/artoolkit.api.js

```diff
@@ -1537,14 +1507,9 @@
 
 			var path = url.split('/').slice(0, -1).join('/')
 			files = files.map(function(file) {
-				// FIXME super kludge - remove it
```

Comment:
If you look into artoolkit.min.js this part of the code does not exist. Maybe he did some changes in the artoolkit.api and not rebuild the artoolkit lib?

Reply:
to be clear: I was refering to the code you deleted ( the red one)

Reply:
I have not rebuilt artoolkit.min.js.
I refer to the initial artoolkit.js and its modification history to modify the file. The code I cleared is the one left by the file modification debugging process. Jeromeetienne made changes over and over again, but in fact just modified ARController.prototype._copyImageToHeap , I just compressed and replaced this part of the code into the min version.

Reply:
It should be rebuilt so that ASM's support is available.
But it is actually unnecessary, because the original ASM code can process the video.